From 37b3c73d9307d1de3b78e3ccba0ba6ba0867d6b8 Mon Sep 17 00:00:00 2001
From: Rui Miguel Silva <rui.silva@linaro.org>
Date: Thu, 23 Feb 2023 10:32:04 +0000
Subject: [PATCH 41/43] nvmxip: move header to include

Move header to include to allow external code
to get the internal bdev structures to access
block device operations.

as at it, just add the UCLASS_NVMXIP string
so we get the correct output in partitions
listing.

Upstream-Status: Pending
Signed-off-by: Rui Miguel Silva <rui.silva@linaro.org>
---
 disk/part.c                          | 3 +++
 {drivers/nvmxip => include}/nvmxip.h | 0
 2 files changed, 3 insertions(+)
 rename {drivers/nvmxip => include}/nvmxip.h (100%)

diff --git a/disk/part.c b/disk/part.c
index 5ee60a7fb591..593dd0004fa4 100644
--- a/disk/part.c
+++ b/disk/part.c
@@ -270,6 +270,9 @@ static void print_part_header(const char *type, struct blk_desc *dev_desc)
 	case UCLASS_NVME:
 		puts ("NVMe");
 		break;
+	case UCLASS_NVMXIP:
+		puts ("NVMXIP");
+		break;
 	case UCLASS_PVBLOCK:
 		puts("PV BLOCK");
 		break;
diff --git a/drivers/nvmxip/nvmxip.h b/include/nvmxip.h
similarity index 100%
rename from drivers/nvmxip/nvmxip.h
rename to include/nvmxip.h
-- 
2.39.2

